PythonAPI/setup.py

```diff
@@ -1,6 +1,4 @@
-from distutils.core import setup
-from Cython.Build import cythonize
-from distutils.extension import Extension
+from setuptools import setup, Extension
 import numpy as np
```

Comment:
This line breaks install for me. I guess `setup_requires=['numpy']` is required below?